experimental/emulated_vault/opt/emulated-vault/vault-server

```diff
@@ -0,0 +1,249 @@
+#! /usr/bin/python
+
+#
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+#
+#
+
+import json
+import logging
+import sys
+import os
+import inspect
+from BaseHTTPServer import HTTPServer
+from BaseHTTPServer import BaseHTTPRequestHandler
+import ssl
+import urlparse, json
```

Comment:
urlparse no longer exists, I think you want [`urllib.parse`](https://docs.python.org/3/library/urllib.parse.html#module-urllib.parse) instead.